### DISCUSSION TITLE
ip type preventing log parsing

### DISCUSSION THREAD

**Discussion by wollsi** [GilbN/geoip2influx#8] [General]
I have a general question about the following line of code:

https://github.com/GilbN/geoip2influx/blob/5cf5a65ebd82bd851034c61c6ea876af0b10833a/geoip2influx.py#L199

It took my some time to figure out why nothing happened after the log `INFO :: Starting log parsing`  appeared and it  seems to be the condition 
```python
ipadd(ip).iptype() == 'PUBLIC'
```
as in my case `ipadd(ip).iptype()` was always `ALLOCATED RIPE NCC`.

I took some further steps and found out that this was the result of my websites only beeing accessible via IPv6 and according to 

https://github.com/autocracy/python-ipy/blob/5ef3b5d92ae2fab56f561ead37cffc9db31000c8/IPy.py#L38-L119

there is no `PUBLIC` for IPv6.

So my actual question is: Is this on purpose, or did noone else have this issue?

**Comment by GilbN** [comment]
Thanks for the PR! The code was based on another repo written in python2. I guess almost no one else had this issue :)
